tests/unit/test_runtime_build.py

```diff
@@ -54,15 +54,12 @@ def _check_source_code_in_dir(temp_dir):
     # check the source file is the same as the current code base
     assert os.path.exists(os.path.join(code_dir, 'pyproject.toml'))
 
-    # The source code should only include the `openhands` folder, but not the other folders
+    # The source code should only include the `openhands` folder,
+    # and pyproject.toml & poetry.lock that are needed to build the runtime image
     assert set(os.listdir(code_dir)) == {
-        'agenthub',
```

Comment:
any idea why we get rid of `agenthub`? would `openhands` package still able to function when that happens?

Reply:
Yes it still functions. You can also import https://pypi.org/project/openhands-ai-test/ and run an agent to verify.